ui/apps/app-center/src/index.tsx

```diff
@@ -28,7 +28,7 @@ export const register: (opts: IntegrationRegistrationOptions) => IAppConfig = op
     label: 'Integration Center',
     area: [MenuItemAreaType.QuickAccessArea, MenuItemAreaType.AppArea],
     logo: { type: LogoTypeSource.ICON, value: 'appCenter' },
-    route: rootRoute,
+    route: routes[EXPLORE],
```

Comment:
currently there is nothing showing on the root route as the default page on IC is the explore page. so it makes sense to have same as the route the sidebar will navigate to

Reply:
keep the exposed route the rootRoute, the app itself will handle internal redirects (in this case redirecting to the explore page). In case you need the app's root route to be used differently it's good to expose it here, and not a subroute